content/docs/getting-started.md

```diff
@@ -0,0 +1,101 @@
+---
+id: getting-started
+title: Getting Started
+permalink: docs/getting-started.html
+next: add-react-to-a-website.html
+redirect_from:
+  - "docs/"
+  - "docs/index.html"
+  - "docs/getting-started-ko-KR.html"
+  - "docs/getting-started-zh-CN.html"
+  - "docs/installation.html"
+  - "download.html"
+  - "downloads.html"
+  - "docs/try-react.html"
+  - "docs/tooling-integration.html"
+  - "docs/package-management.html"
+  - "docs/language-tooling.html"
+  - "docs/environments.html"
+---
+
+This page is an overview of the React documentation and related resources.
+
+- [Try React](#try-react)
+- [Learn React](#learn-react)
+- [Staying Informed](#staying-informed)
+- [Versioned Documentation](#versioned-documentation)
+- [Something Missing?](#something-missing)
+
+## Try React
+
+React is designed for gradual adoption, and you can use as little or as much React as you need. Whether you want to get a taste of React, add some interactivity to a simple HTML page, or start a complex React-powered app, the links in this section will help you get started.
+
+### Online Playgrounds
+
+If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world) or [CodeSandbox](https://codesandbox.io/s/new). If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so don't use it in production.
```

Comment:
I might walk back "don't use" slightly since there are a lot of toys that can be used like this – maybe like "so it's not the best choice when performance matters"? or "so we'd only recommend using this for simple demos"?